cmd/influx2cortex/main.go

```diff
@@ -40,7 +42,14 @@ func Run() error {
 		return err
 	}
 
-	api, err := influx.NewAPI(logger, apiConfig)
+	remoteWriteRecorder := remotewrite.NewRecorder("influx", prometheus.DefaultRegisterer)
+	client, err := remotewrite.NewClient(remoteWriteConfig, remoteWriteRecorder, nil)
+	if err != nil {
+		level.Error(logger).Log("msg", "Failed to instantiate remotewrite.API for influx2cortex", "err", err)
```

Comment:
```suggestion
		level.Error(logger).Log("msg", "failed to instantiate remotewrite.API for influx2cortex", "err", err)
```

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

> Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context. That is, use fmt.Errorf("something bad") not fmt.Errorf("Something bad"), so that log.Printf("Reading %s: %v", filename, err) formats without a spurious capital letter mid-message. This does not apply to logging, which is implicitly line-oriented and not combined inside other messages.